pkg/controllers/state/suite_test.go

```diff
@@ -880,16 +886,7 @@ var _ = Describe("Node Resource Level", func() {
 				},
 			},
 		})
-		node := test.Node(test.NodeOptions{
-			ObjectMeta: metav1.ObjectMeta{
-				Labels: map[string]string{
-					v1.NodePoolLabelKey:            nodePool.Name,
-					corev1.LabelInstanceTypeStable: cloudProvider.InstanceTypes[0].Name,
-				},
-				Finalizers: []string{v1.TerminationFinalizer},
-			},
-			ProviderID: nodeClaim.Status.ProviderID,
-		})
+		node := test.NodeClaimLinkedNode(nodeClaim)
```

Comment:
I wonder if we need a function to ensure that we mark that the NodeClaim has scheduled so that you don't modify the existing function logic

Reply:
I'm not sure I follow. I changed this to ensure the Node has the NodeClass label on it for the management check, I don't think it makes sense to require the use of the lifecycle controller in any test that needs to apply that label.